.github/workflows/lower-bound-requirements.yml

```
@@ -0,0 +1,36 @@
+name: Minimum supported dependencies
+
+on:
+  # Run weekly at 1:23 UTC
+  schedule:
+    - cron: "23 1 * * 0"
```

It might make sense to run leading-edge ("running at head") tests on a regular basis, regardless of any PR, but the lower-bound makes more sense to run only when there are changes in the code. New versions of dependencies can't break the lower-bound, but adding to the codebase can.

```suggestion
  pull_request:
  push:
    branches:
      - main
```